projects/ngrx.io/content/guide/data/entity-change-tracker.md

```diff
@@ -269,24 +278,69 @@ is reclassified as "deleted".
 Its `originalValue` stays the same.
 Undoing the change will restore the entity to the collection in its pre-update state.
 
-<a id="enable-change-tracking"></a>
+<a id="merge-strategy"></a>
+
+### Merge Strategies
+
+You can determine how NgRx will merge entities after query or save or a cache operation by setting the
+entity action's optional `mergeStrategy` property to one of the three strategy enums:
+
+1. `IgnoreChanges` - Update the collection entities and ignore all change tracking for this operation. The ChangeState is untouched.
+
+2. `PreserveChanges` -  Updates current values for unchanged entities. 
+If entities are changed, preserves their current values and overwrites their originalValues with the merge entity.
```

Comment:
```suggestion
If entities are changed, preserves their current values and overwrites their original values with the merged entity.
```

Reply:
Thanks. Rewrote these sentences for clarity.